test/chainedvector.jl

```diff
@@ -284,6 +284,17 @@
     @test all(==(2), x)
     @test length(x) == 5
 
+    # https://github.com/JuliaData/SentinelArrays.jl/issues/97
+    x = ChainedVector([[18, 70, 92, 15, 65], [25, 14, 95, 54, 57]])
+    @test findmax(x) == (95, 8)
+    @test findmin(x) == (14, 7)
+    @test argmax(x) == 8
+    @test argmin(x) == 7
+    @test findmax(inv, x) == (inv(14), 7)
+    @test findmin(inv, x) == (inv(95), 8)
+    @test argmax(inv, x) == 14
+    @test argmin(inv, x) == 95
```

Comment:
```suggestion
    @test let (val, idx) = findmax(x)
         max_val = maximum(x)
         val == max_val == x[idx]
    end
    @test let (val, idx) = findmin(x)
         min_val = minimum(x)
         val == min_val == x[idx]
    end
    @test x[argmax(x)] == maximum(x)
    @test x[argmin(x)] == minimum(x)
    @test let (val, idx) = findmax(inv, x)
         max_val = maximum(inv, x)
         val == max_val == inv(x[idx])
    end
    @test let (val, idx) = findmin(inv, x)
         min_val = minimum(inv, x)
         val == min_val == inv(x[idx])
    end
    @test x[argmax(inv, x)] == maximum(inv, x)
    @test x[argmin(inv, x)] == minimum(inv, x)
```

IMO writing it like this would be a bit more robust against changing `x` in the future and forgetting to change the hardcoded data. This also ends up testing that `argmax` actually has to return an index, and not just the right number for this test case :)

Reply:
I'll adopt this in #99